lib/python/ray/services.py

```diff
@@ -134,5 +136,5 @@ def start_singlenode_cluster(return_drivers=False, num_objstores=1, num_workers_
     time.sleep(0.5)
     return driver_workers
   else:
-    ray.connect(scheduler_address, objstore_addresses[0], address(IP_ADDRESS, new_worker_port()), print_task_info=print_task_info)
+    ray.connect(scheduler_address, objstore_addresses[0], address(IP_ADDRESS, new_worker_port()), mode=driver_mode)
```

Comment:
I'm thinking of providing two kinds of modes for drivers

- ray.DRIVER_MODE for when you run a script
- ray.SHELL_MODE for when you use Ray interactively in a shell

The idea being that ray.SHELL_MODE should be a little more verbose. Still experimenting with this though.

There's also ray.WORKER_MODE, which only logs to the log file.